projects/components/src/action-menu/action-menu.component.ts

```diff
@@ -249,6 +233,32 @@ export class ActionMenuComponent<R, T> {
      */
     isObservable = isObservable;
 
+    /**
+     * The list of entities selected on which contextualActions are performed. As they are also used for calculating the
+     * availability of actions, action lists are updated when the input is updated
+     * @param val Is an array in case of batch selection and is a single item for example in the case of data grids with single selection
+     */
+    @Input() set selectedEntities(val: R[] | R) {
+        if (!Array.isArray(val)) {
+            val = val ? [val] : [];
+        }
+        // Shallow equal does the job for most of the cases and currently saves a lot of calculations
+        if (this.isShallowEqual(val, this._selectedEntities)) {
+            return;
+        }
+        this._selectedEntities = val;
+        this.updateDisplayedActions();
+    }
+    private _selectedEntities: R[] = [];
+
+    /**
+     * This not a getter because getter always returns an array while the setter above can accept an array or a single
```

Comment:
Returns the selected entities. Note: this is not a getter because its matching setter can accept an array or a single item but this method always returns an array.